lib/ui/views/app_selector/app_selector_viewmodel.dart

```diff
@@ -69,6 +70,36 @@ class AppSelectorViewModel extends BaseViewModel {
     return true;
   }
 
+  Future<void> searchSuggestedVersionOnWeb(BuildContext context,
+      {required String packageName, required String appName, bool isInstalled = false,}) async {
+    final String search = FlutterI18n.translate(
+      context,
+      'appSelectorCard.search',
+    );
+    final String apk = FlutterI18n.translate(
+      context,
+      'appSelectorCard.apk',
+    );
+    final String version = FlutterI18n.translate(
+      context,
+      'appSelectorCard.version',
+    );
+    final String suggestedVersion = getSuggestedVersion(packageName);
+    String stringUrl;
+    if(isInstalled){
+      stringUrl = 'https://www.google.com/$search?q=$appName'
+          '+$apk';
+    }else{
+      stringUrl = 'https://www.google.com/$search?q=$appName'
+          '+$apk+$version+v+$suggestedVersion';
+    }
```

Comment:
This doesn't seem right to me, why would having the app installed effect the need to include the suggested version?

Reply:
@Ushie, Honestly, I can't remember why i added this. Will remove it.